src/sinks/datadog/logs/mod.rs

```diff
@@ -32,6 +32,8 @@ pub(crate) use config::DatadogLogsConfig;
 
 use crate::config::SinkDescription;
 
+pub(crate) const NAME: &str = "datadog_logs";
```

Comment:
😛 you love these consts

Reply:
yeah... soon as I have to type "datadog_logs" and I know it's already in the source at least once- it's automatic.